cypress/integration/pages/frontPage/testsForCanonicalOnly.js

```diff
@@ -11,6 +11,28 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({ service }) => {
   if (Cypress.env('APP_ENV') === 'local') {
     runCanonicalAdsTests({ service });
   }
+  describe(`No testsToAlwaysRunForCanonicalOnly to run for ${service}`, () => {
+    describe(`US Election Banner`, () => {
+      it('should have US Election banner for Hindi, Arabic, Portuguese, Mundo, Persian, Russian', () => {
```

Comment:
`should have US Election banner if toggle is enabled`

Reply:
Just learned about the toggle. Will change :)